hash/gba.xml

```diff
@@ -7032,6 +7109,19 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<!-- Pitch demo for a GBA port. -->
+	<software name="demonscrest">
+		<!-- The patch doesn't seem to be needed in MAME, but on other emulators, you have to patch 0x122384 with 0xc046c046. -->
+		<description>Demon's Crest (prototype)</description>
+		<year>????</year>
+		<publisher>Rival Dreams</publisher>
```

Comment:
Can you at least give this an estimated decade or century?  I’d blindly guess it’s likely `<year>200?</year>`.

Reply:
Done. I changed this to "2002?" since it's seemingly built from Dark Eden.